pkg/sentry/platform/ring0/lib_arm64.go

```diff
@@ -16,10 +16,27 @@
 
 package ring0
 
-// LoadFloatingPoint loads floating point state by the most efficient mechanism
-// available (set by Init).
-var LoadFloatingPoint func(*byte)
+// CPACR_EL1, Architectural Feature Access Control Register.
```

Comment:
Doc comments should be a sentence starting with the name of the function. e.g., "CPACREL1 returns the value of the CPACR_EL1 register."

https://golang.org/doc/effective_go.html#commentary

Reply:
Done.